rapids-4-spark-tools/README.md

```diff
@@ -1,78 +1,197 @@
 # Spark Qualification and Profiling tools
 
-The qualification tool is used to rank a set of applications to determine if the RAPIDS Accelerator for Apache Spark is a good fit for those applications.
+The qualification tool is used to rank a set of applications to determine if the RAPIDS Accelerator for Apache Spark 
+is a good fit for those applications. Works with CPU generated event logs.
 
 The profiling tool generates information which can be used for ranking applications debugging and profiling.
-Information such as Spark version, executor information, Properties and so on.  Works with both cpu and gpu generated event logs.
+Information such as Spark version, executor information, properties and so on.  
+Works with both CPU and GPU generated event logs.
```

Comment:
qualification tool is really mean to be run on cpu logs.

Reply:
Yep this is for profiling tool part. 

In the qualification part, we mentioned `Works with CPU generated event logs.`

Does it look good to you?